packages/openmrs/src/http.js

```diff
@@ -0,0 +1,46 @@
+import { expandReferences } from '@openfn/language-common/util';
+import * as util from './Utils';
+
+/**
+ * Options object
+ * @typedef {Object} OpenMRSOptions
+ * @property {object} query - An object of query parameters to be encoded into the URL
+ * @property {object} headers - An object of all request headers
+ * @property {object} body - The request body (as JSON)
+ * @property {string} baseUrl - The base url for the request
```

Comment:
We should not document baseUrl here. Technically we should not support it.

The `http.request` function is ALWAYS relative to the baseUrl on config. Users should not be setting this.